domains/krix.json

```diff
@@ -5,7 +5,7 @@
         "email": "kristjan.kocevar89@gmail.com"
       },
     "record": {
-        "A": ["35.186.245.55"],
+        "CNAME": "krix12.github.io",
```

Comment:
MX & CNAME cannot be used together, please consider switching to a redirect or an A record.